### PR TITLE
Misc addons 12.0 odoo backup sh docs upd

### PR DESCRIPTION
**Pull request by em230418**
Based on https://github.com/it-projects-llc/addons-dev/pull/678
Added commits to pass test and fix config reading/writing

### CODE REVIEW

**Comment by em230418** [comment]
Also, compare link to make sure, that I didn't edit @Mirgalimova's commits: https://github.com/Mirgalimova/addons-dev/compare/misc-addons-12.0-odoo_backup_sh-docs_upd...em230418:misc-addons-12.0-odoo_backup_sh-docs_upd

**Comment by em230418** [comment]
I approve to merge it now

File: odoo_backup_sh/__manifest__.py [hunk 1]
```diff
@@ -8,9 +9,9 @@
     "images": ['images/odoo-backup.sh.jpg'],
     "version": "12.0.1.0.0",
 
-    "author": "IT-Projects LLC, Stanislav Krotov",
+    "author": "IT-Projects LLC",
     "support": "apps@it-projects.info",
-    "website": "https://apps.odoo.com/apps/modules/12.0/odoo_backup_sh/",
+    "website": "https://www.it-projects.info/",
```
Comment (yelizariev) [inline odoo_backup_sh/__manifest__.py]: It was correct link. The link will work once we publish the module

File: odoo_backup_sh/__manifest__.py [hunk 2]
```diff
@@ -39,5 +40,4 @@
     "auto_install": False,
     "installable": True,
     "application": True,
-    "uninstall_hook": "uninstall_hook",
```
Comment (yelizariev) [inline odoo_backup_sh/__manifest__.py]: Why it's deleted?

**Comment by yelizariev** [comment]
@em230418 you should not base your branch on #678 because that PR has doc updates only which are not finished. But we can skip that for now

File: odoo_backup_sh/__manifest__.py (hunk 2 above)
Comment (em230418) [reply odoo_backup_sh/__manifest__.py]: There is already uninstall_hook key. Line 38

File: odoo_backup_sh/__manifest__.py (hunk 1 above)
Comment (em230418) [reply odoo_backup_sh/__manifest__.py]: Returned back

File: odoo_backup_sh/__manifest__.py (hunk 2 above)
Comment (yelizariev) [reply odoo_backup_sh/__manifest__.py]: Then you need to remove the one with None value

Comment (em230418) [reply odoo_backup_sh/__manifest__.py]: Done

**Comment by em230418** [comment]
I approve to merge it now

**Comment by yelizariev** [comment]
I approve to merge it now

**Comment by yelizariev** [comment]
I approve to merge it now

**Pull request merged**